worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java

```diff
@@ -62,13 +67,29 @@
 
     private final ContractRepository contractRepository;
 
+    @Value("${eob.job.patient.number.per.thread}")
```

Comment:
I almost feel like you need to create a class with all of these parameters to clean things up because there are a lot of variables here to not have some sort of class containing it.

Reply:
Will do